From patches Mon Sep 17 00:00:00 2001
From: Peter Maydell <peter.maydell@linaro.org>
To: Peter Maydell <peter.maydell@linaro.org>, qemu-devel@nongnu.org
Cc: Anthony Liguori <aliguori@us.ibm.com>, Riku Voipio <riku.voipio@iki.fi>,
 patches@linaro.org, qemu-trivial@nongnu.org, qemu-devel@nongnu.org,
 qemu-trivial@nongnue.org
Date: Sun, 21 Apr 2013 13:30:03 +0100
Message-id: 1366547403-16480-1-git-send-email-peter.maydell@linaro.org
Subject: [PATCH 1/1] linux-user: fix compile error due to stray colon at end
 of #ifdef line

Remove a stray colon from the end of a #ifdef line. Some versions
of gcc complain about this:
 linux-user/syscall.c: In function ‘do_syscall’:
 linux-user/syscall.c:7606:28: error: extra tokens at end of #ifdef directive [-Werror]

Signed-off-by: Peter Maydell <peter.maydell@linaro.org>
---
Obviously this is 1.5 material; don't care whether it goes through
linux-user or qemu-trivial or direct commit.

 linux-user/syscall.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/linux-user/syscall.c b/linux-user/syscall.c
index c705960..30e93bc 100644
--- a/linux-user/syscall.c
+++ b/linux-user/syscall.c
@@ -7603,7 +7603,7 @@ abi_long do_syscall(void *cpu_env, int num, abi_long arg1,
 #endif
 #else
     case TARGET_NR_sendfile:
-#ifdef TARGET_NR_sendfile64:
+#ifdef TARGET_NR_sendfile64
     case TARGET_NR_sendfile64:
 #endif
         goto unimplemented;
-- 
1.7.11.4

